internal/cephfs/util/util.go

```diff
@@ -38,12 +36,6 @@ const (
 )
 
 func ParseTime(ctx context.Context, createTime time.Time) (*timestamp.Timestamp, error) {
-	tm, err := ptypes.TimestampProto(createTime)
-	if err != nil {
-		log.ErrorLog(ctx, "failed to convert time %s %v", createTime, err)
-
-		return tm, err
-	}
-
-	return tm, nil
+	// todo : we can remove the signature to not have error return
```

Comment:
This is a very small change address it now instead of adding a TODO and addressing it later.

Reply:
@Madhu-1 ptal. tx